client/src/components/add-job-post-component/job-info-form/index.jsx

```diff
@@ -0,0 +1,252 @@
+
+        {/* Compensation Rate Field */}
+        <FormControl fullWidth sx={{ m: 1, width: "47%" }}>
+          <InputLabel id="compensationRateLabel">Compensation rate</InputLabel>
+          <Select
+            sx={{ textAlign: "left" }}
+            labelId="compensationRateLabel"
+            id="compensationRateLabel"
+            value={jobPostData.rate_of_pay_frequency}
+            label="Compensation rate"
+            onChange={(e) =>
+              handleInputChange(e.target.value, "rate_of_pay_frequency")
+            }
+            required
+            InputLabelProps={{ required: false }}
+          >
+            {COMPENSATION_RATES.map((jobType) => (
+              <MenuItem key={jobType} value={jobType}>
+                {jobType}
+              </MenuItem>
+            ))}
+          </Select>
+          <FormHelperText>*Required</FormHelperText>
+        </FormControl>
+
+        {/* Hours Per Week Field */}
+        <TextField
+          fullWidth
+          sx={{ m: 1, width: "47%" }}
+          id="hoursPerWeek"
+          type="number"
+          label="Hours per week"
+          inputProps={{ min: 0 }}
+          value={jobPostData.hours_per_week}
+          onChange={(e) => {
+            const { value } = e.target;
+            if (/^\d*\.?\d*$/.test(value))
+              handleInputChange(value, "hours_per_week");
+          }}
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Employment Type Field */}
+        <FormControl sx={{ m: 1, width: "96%" }}>
+          <Autocomplete
+            multiple
+            id="tags-outlined"
+            options={JOB_TYPES}
+            getOptionLabel={(option) => option}
+            defaultValue={jobPostData.job_type}
+            filterSelectedOptions
+            onChange={(e, value) => handleInputChange(value, "job_type")}
+            required
+            renderInput={(params) => (
+              <TextField
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...params}
+                label="Employment Type"
+                labelId="employmentTypeLabel"
+                id="employmentTypeLabel"
+              />
+            )}
+          />
+          <FormHelperText>*Required</FormHelperText>
+        </FormControl>
+
+        {/* Creation Date Picker */}
+        <LocalizationProvider dateAdapter={AdapterDayjs}>
+          <DatePicker
+            id="creation"
+            label="Creation Date"
+            fullWidth
+            sx={{ m: 1, width: "47%" }}
+            value={jobPostData.creation_date}
+            disabled
+            renderInput={(params) => (
+              // eslint-disable-next-line
+              <TextField {...params} error={false} helperText="" required />
+            )}
+            required
+            InputLabelProps={{ required: false }}
+          />
+        </LocalizationProvider>
+
+        {/* Expiration Date Picker */}
```

Comment:
The expiration date can be set to before the creation date. I'm not sure if we need to account for this, but I wanted to take note of it just in case.

Reply:
Right now for me the selection for expiration dates are only for today and onwards. Is it different for you?

<img width="799" alt="image" src="https://github.com/user-attachments/assets/0296962b-0f18-43bb-9c1c-eef6a5ba38e1" />

Reply:
Oh I think I might've made a mistake. Sorry!